Alliance.Client/Patch/Behaviors/AllianceAgentVisualSpawnComponent.cs

```diff
@@ -191,7 +191,7 @@ public void SpawnAgentVisualsForPeer(MissionPeer missionPeer, AgentBuildData bui
 			if (missionPeer.IsMine && buildData.AgentVisualsIndex == 0)
 			{
 				// Play random cheer for Uruk because why not
-				if (baseMonsterFromRace.StringId == "uruk")
+				if (baseMonsterFromRace.StringId == "uruk" || baseMonsterFromRace.StringId == "orc")
```

Comment:
I would recommand to store somewhere all possibles races and to use this here (and everywhere you check for races)

Reply:
Yep this is planned for the troll branch. We'll have extensions for Agent to easily check if it's a custom race 👍

Reply:
(we'll be able to do agent.IsUruk / agent.IsTroll etc.)